File: acmicpc/10250/10250.py

def testcase():
    h, w, n = [ int(i) for i in input().split()]
    if n%h:
        print("{}{:02}".format(n%h, n//h + 1))
    else:
        print("{}{:02}".format(h, n//h))

if __name__ == "__main__":
    t = int(input())
    for i in range(t):
        testcase()

File: acmicpc/10250/10250.cc
#include <cstdio>

void testcase() {
    int h,w,n;
    scanf("%d %d %d", &h, &w, &n);

    if (n % h) {
        printf("%d%02d\n", n%h, n/h+1);
    } else {
        printf("%d%02d\n", h, n/h);
    }
}
int main() {
    int t;
    scanf("%d", &t);
    for (;t--;) {
        testcase();
    }
}

